janis_bioinformatics/tools/common/indexfasta.py

```diff
@@ -22,16 +69,44 @@ def constructor(self):
 
         self.input("reference", Fasta)
 
-        self.step("create_bwa", BwaIndexLatest(reference=self.reference))
+        # Change the default BWA index algorithm to bwtsw (for human genome), and up blockSize to 50M
+        self.input("bwa_algorithm", String(optional=True), default="bwtsw")
+        self.input("bwa_block_size", Int(optional=True), default=50 * 1000000)
```

Comment:
Possibly easier to read `50 * 1000 * 1000` : )

Reply:
Changed to `int(5E7)`.